io.github.quantum_mutnauq.fast_reader_gtk.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.github.quantum_mutnauq.fast_reader_gtk
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: fast-reader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=wayland
+  
+modules:
+  - name: libconfig
+    buildsystem: simple
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
```

Comment:
this should be `buildsystem: autotools` instead.

Reply:
The build isn't working; it fails with an error stating that Makefile.in could not be found.

Reply:
if you use the tarball then there is no problem. Otherwise add the autoreconf command to a `shell` source.

Reply:
The tar file for version 1.5 is not available. How can I include the shell source?